requirements.txt

```diff
@@ -25,3 +25,5 @@ redis==3.5.3
 gunicorn==20.1.0
 gevent==20.9.0
 pyparsing==2.4.7
+Flask-SQLAlchemy==2.5.1
+SQLAlchemy==1.4.18
```

Comment:
we are aiming to get rid of it completely

Reply:
mysqlclient is still needed, as sqlalchemy relies on it as it's default driver for mysql and mariadb databases. Alternatively, it could be replaced by one of the other supported drivers listed here https://docs.sqlalchemy.org/en/14/dialects/mysql.html .